### PR TITLE
Documentation added for Results layout

### PR DESCRIPTION
**Pull request by yerramshilpa**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/documentation/src/lib/pages/layout/layout.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<div class="n-sds-page margin-top-2">
```
Comment (davereed) [inline libs/documentation/src/lib/pages/layout/layout.component.html:1]: n-sds-page should be sds-page

File: libs/documentation/src/lib/pages/layout/layout.component.ts [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    defaultSortValue: 'relevanceDescending',
+    pageSize: 25,
+    sortList: [
+      { text: 'Relevance', value: 'relevanceDescending' },
+      { text: 'Entity Name', value: 'legalBusinessName' },
+      { text: 'Status', value: 'registrationStatus' }
+    ]
+  };
+
+  public fields: FormlyFieldConfig[] = [
```
Comment (davereed) [inline libs/documentation/src/lib/pages/layout/layout.component.ts:35]: Can we move to the structure that Christy uses, she has a service (and a base service sometimes) that handles the layout config, filters, model, etc.

File: libs/documentation/src/lib/pages/layout/layout.component.ts [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    {
+      key: 'keyword',
+      wrappers: ['filterwrapper'],
```
Comment (davereed) [inline libs/documentation/src/lib/pages/layout/layout.component.ts:38]: Remove all wrappers, prefer 'group' template options.

File: libs/documentation/src/lib/pages/layout/layout.component.html (hunk 1 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/pages/layout/layout.component.html]: Removed

File: libs/documentation/src/lib/pages/layout/layout.component.ts (hunk 2 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/pages/layout/layout.component.ts]: Updated to one service to make it simple

File: libs/documentation/src/lib/pages/layout/layout.component.ts (hunk 3 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/pages/layout/layout.component.ts]: Updated to group

**Pull request merged**